tests/test_fyle_integrations_imports/test_modules/test_webhook_attributes.py

```
@@ -142,6 +144,23 @@ def test_process_webhook_corporate_card(db):
+    workspace_id = 1
+    processor = WebhookAttributeProcessor(workspace_id)
+
+    with mock.patch('apps.mappings.helpers.patch_corporate_card_integration_settings') as mock_patch:
+        processor.process_webhook(webhook_payloads['corporate_card_created'])
+        mock_patch.assert_called_once_with(workspace_id=workspace_id)
+
+    with mock.patch('apps.mappings.helpers.patch_corporate_card_integration_settings') as mock_patch:
+        mock_patch.side_effect = Exception("Test error")
+        processor.process_webhook(webhook_payloads['corporate_card_created'])
+        assert ExpenseAttribute.objects.filter(workspace_id=workspace_id, attribute_type='CORPORATE_CARD',
+                                              source_id='card_123').exists()
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)